src/App.vue

```diff
@@ -218,8 +234,29 @@ export default class App extends Mixins(mixins.TransactionMixin, NodeErrorMixin)
 
   private setResponsiveClassDebounced = debounce(this.setResponsiveClass, 250);
 
+  handleLocalStorageChange() {
+    const usagePercentage = calculateStorageUsagePercentage();
+
+    console.info(`Current localStorage usage: ${usagePercentage.toFixed(2)}%`);
+    if (usagePercentage >= LOCAL_STORAGE_LIMIT_PERCENTAGE) {
+      this.showErrorLocalStorageExceed = true;
+    }
+  }
+
+  clearLocalStorage() {
```

Comment:
This method has not component dependencies - maybe is better to transfer it to `src/utils/storage` ?